libr/cons/dietline.c

```diff
@@ -1285,7 +1285,8 @@ R_API const char *r_line_readline_cb(RLineReadCallback cb, void *user) {
 		__print_prompt ();
 	}
 	r_cons_break_push (NULL, NULL);
-	r_cons_enable_mouse (I.hud);
+	// r_cons_enable_mouse (I.hud);
+	r_cons_enable_mouse (false);
```

Comment:
I.hud is a struct, i so the commented line was related to the vi mode, which seems wrong to me. so I would probably say that its fine to remove the commented line.